std/range/package.d

```diff
@@ -9223,3 +9223,5 @@ if (Rs.length > 1 &&
     m.popFront;
     assert(m.empty);
 }
+
+package T* addressOf(T)(ref T val) { return &val; }
```

Comment:
It's necessary to take an address of returned reference from getter property.

1. After the property enforcement finished, a getter property cannot be called with parenthesis: `prop()` -> `prop`
2. But applying address operator `&` on property function name returns the address of "property function": `&prop` -> function pointer or delegate

I had proposed adding a special syntax difference between `&prop` and `&(prop)`, but it was rejected.
So, I think we should use the helper function like `addressOf(prop)` or `prop.addressOf` for the purpose.